src/common/types/cpuArchitecture.ts

```diff
@@ -53,7 +53,7 @@ export const getSupportedCpuArchitectures = (
       ArchitectureSupportLevelId,
       SupportLevel,
     ][]) {
-      if (supportLevel !== 'unsupported') {
+      if (supportLevel !== 'unsupported' && supportLevel !== 'unavailable') {
```

Comment:
Could we try to group this into a function that looks at the "not supported" values? There are less changes to forget about either of these states in that way.

Are these two values always going to mean the same to the UI?

Reply:
At this moment yes.
Later maybe the behavior will be different